src/main/java/org/apache/pirk/querier/wideskies/QuerierDriverCLI.java

```diff
@@ -239,12 +242,19 @@ private boolean parseOptions()
       }
       SystemConfiguration.setProperty(CERTAINTY, getOptionValue(CERTAINTY));
 
-      if (!hasOption(QUERYNAME))
+      if (!hasOption(QUERYID))
       {
-        logger.info("Must have the option " + QUERYNAME);
+        logger.info("Must have the option " + QUERYID);
         return false;
       }
-      SystemConfiguration.setProperty(QUERYNAME, getOptionValue(QUERYNAME));
+      
+      // create a formatted date/time string
+      Date date = new Date();
+      SimpleDateFormat sdf = new SimpleDateFormat("MM.dd.yyyy_hh.mm.ss");
+      String formattedDate = sdf.format(date);
```

Comment:
According to: https://pirk.incubator.apache.org/for_users, the required JDK is 1.7.  I'm pro 1.8, but that's not the subject of this PR.  Was the decision made to drop 1.7 support?